pkg/task/common.go

```diff
@@ -120,8 +122,10 @@ type Config struct {
 	// should be removed after TiDB upgrades the BR dependency.
 	Filter filter.MySQLReplicationRules
 
-	TableFilter        filter.Filter `json:"-" toml:"-"`
-	CheckRequirements  bool          `json:"check-requirements" toml:"check-requirements"`
+	TableFilter       filter.Filter `json:"-" toml:"-"`
+	CheckRequirements bool          `json:"check-requirements" toml:"check-requirements"`
+	// EnableOpentracing is whether to enable opentracing
+	EnableOpentracing  bool          `json:"enable-opentracing" toml:"enable-opentracing"`
```

Comment:
🤔 [the proper spelling](https://opentracing.io/) is OpenTracing with a capital T. what is the name of this flag in other PingCAP programs?

Reply:
https://github.com/pingcap/tidb/blob/master/config/config.go#L125
https://github.com/pingcap/tidb/blob/master/config/config.go#L475
I pick these two configurations from TiDB and combine them into `enable-opentracing` here 😿
Any better idea?

Reply:
ok. call the variable `EnableOpenTracing` and the parameter `--enable-opentracing` or `--opentracing`.